.github/CODEOWNERS

```diff
@@ -58,10 +58,10 @@
 /.github/workflows/serverless-binary-size.yml        @DataDog/serverless @Datadog/serverless-aws
 /.github/workflows/serverless-integration.yml        @DataDog/serverless @Datadog/serverless-aws
 /.github/workflows/cws-btfhub-sync.yml               @DataDog/agent-security
-/.github/workflows/gohai.yml                         @DataDog/agent-shared-components
-/.github/workflows/go-update-commenter.yml           @DataDog/agent-shared-components
-/.github/workflows/update_dependencies.yml           @DataDog/agent-shared-components
-/.github/workflows/buildimages-update.yml            @DataDog/agent-delivery @DataDog/agent-shared-components
+/.github/workflows/gohai.yml                         @DataDog/agent-runtimes
```

Comment:
I think Gohai should be owned by Agent Configuration if you also own the metadata ?

Reply:
Probably would +1 this as well since `gohai` input is pretty much solely used for the metadata I believe. if that changes in the future, I can see the need for ownership to change though.